homeassistant/components/homematicip_cloud/config_flow.py

```diff
@@ -65,9 +65,9 @@ async def async_step_link(self, user_input=None) -> FlowResult:
             if authtoken:
                 _LOGGER.info("Write config entry for HomematicIP Cloud")
                 return self.async_create_entry(
-                    title=self.auth.config.get(HMIPC_HAPID),
+                    title=self.auth.config[HMIPC_HAPID],
                     data={
-                        HMIPC_HAPID: self.auth.config.get(HMIPC_HAPID),
+                        HMIPC_HAPID: self.auth.config[HMIPC_HAPID],
```

Comment:
Based on the rest of the config flow, it seems to me that `HMIPC_HAPID` is a compulsory key.